resources/food/tomato.tscn

```diff
@@ -13,6 +13,8 @@ collision_mask = 8519687
 mass = 0.2
 center_of_mass_mode = 1
 center_of_mass = Vector3(0, 0.06, 0)
+freeze = true
```

Comment:
why specifically the potato and tomato. Isn't that something that might apply to every food item?

Reply:
Well yes

Reply:
Then it belongs to Food.tscn doesn't it?